ports/onnxruntime/portfile.cmake

```
@@ -0,0 +1,365 @@
+# Only limited triplets supported
+# x64-windows: x64-windows-static-md
+# x64-linux
+# No support for osx and uwp OS
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_check_linkage(ONLY_STATIC_LIBRARY) 
+    vcpkg_fail_port_install(ON_ARCH "arm" "arm64" "wasm32" "x86" ON_TARGET "uwp")
+elseif(VCPKG_TARGET_IS_LINUX)
+    vcpkg_fail_port_install(ON_ARCH "x86")
+else()
+    vcpkg_fail_port_install(ALWAYS)
+endif()
+
+set(ORT_REVISION "v1.5.3")
+
+vcpkg_find_acquire_program(GIT)
+get_filename_component(GIT_PATH ${GIT} DIRECTORY)
+vcpkg_find_acquire_program(PYTHON3)
+get_filename_component(PYTHON3_PATH ${PYTHON3} DIRECTORY)
+vcpkg_add_to_path(${PYTHON3_PATH})
+vcpkg_find_acquire_program(NINJA)
+get_filename_component(NINJA_PATH ${NINJA} DIRECTORY)
+vcpkg_add_to_path(${PYTHON3_PATH})
```

```suggestion
vcpkg_add_to_path(${NINJA_PATH})
```
This might be `NINJA_PATH`.

---

correct.